tests/cases/compiler/jsxImportInAttribute.tsx

```diff
@@ -0,0 +1,14 @@
+//@jsx: preserve
+//@module: commonjs
+
+//@filename: component.d.ts
+declare module "Test" {
+	export default class Text { }

```

Comment:
For the future, can you set your user settings to `"translate_tabs_to_spaces": true`?